<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>931</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>931</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 179, 179);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>881</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>LC3 simulator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>263</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="memory_lable">
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>261</width>
         <height>101</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; font-style:italic; color:#ffffff;&quot;&gt;Memory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="memory_table">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(159, 159, 159);
border-color: rgb(0, 0, 0);
border-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>122</y>
      <width>553</width>
      <height>406</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>553</width>
      <height>406</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>553</width>
      <height>406</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="3">
          <widget class="QLabel" name="label_MDR">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_R0">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_P">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_R1">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_N">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_R2">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_MAR">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_R3">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_IR">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_Z">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_Pc">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="phases_lable">
           <property name="minimumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>132</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>instruction</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_R4">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_R5">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_R6">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_R7">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="Start_Assemble_PushButton">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.512, y1:0.352273, x2:1, y2:1, stop:0.173709 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Assemble</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Next_PushButto">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.512, y1:0.352273, x2:1, y2:1, stop:0.173709 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="StopPushButto">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>51</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.512, y1:0.352273, x2:1, y2:1, stop:0.173709 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Upload_file_pushButton">
       <property name="minimumSize">
        <size>
         <width>551</width>
         <height>141</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0.512, y1:0.352273, x2:1, y2:1, stop:0.173709 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Click hear for upload your file  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
